typo fixed in sigfastblock.2

provomittt opened the pull request:
In line 132: `successfull`-> `successful`

This is from the Advanced UNIX Programming Course (Fall’23) at NTHU.

lwhsu commented:
Please check the instruction from TA. You don't have to provide your name and email in the comment but you have to provide those in the commit metadata.

As you can see in https://patch-diff.githubusercontent.com/raw/freebsd/freebsd-src/pull/1001.patch , the `From:` (`Author:` in git) line is generated by github, which is not encouraged.

Also, please use:

```
Event: Advanced UNIX Programming Course (Fall’23) at NTHU
```

At the bottom of the commit message.

provomittt commented:
Thanks for the reply! I've updated the commit message and edited the comment.

lwhsu commented:
https://patch-diff.githubusercontent.com/raw/freebsd/freebsd-src/pull/1001.patch still seems different than what you want to use. Also, please squash the commits -- we don't need the duplicate fixes and the merge commit.

provomittt commented:
Thank you for helping!
I squashed the duplicated commits, removed the merge commit, deleted personal information in the commit message, and updated the author settings.

lwhsu commented:
Please add an extra blank line before `Event:` in the commit message.

lwhsu commented:
Was fixed in a70008e95b907336bddbcc2aa9d567df6e3a41b6
